lib/core/src/sdk.rs

```diff
@@ -56,14 +56,21 @@ pub struct LiquidSdk {
 
 impl LiquidSdk {
     pub async fn connect(req: ConnectRequest) -> Result<Arc<LiquidSdk>> {
-        let config = req.config;
-        let sdk = LiquidSdk::new(config, req.mnemonic)?;
+        let breez_server = BreezServer::new("https://bs1.breez.technology:443".into(), None)?;
+        let swapper_urls = breez_server.fetch_boltz_swapper_urls().await?;
```

Comment:
Perhaps silent handle errors here? I am thinking in case the user is offline this will return error on restart.

Reply:
Done: 7bcc63b90a3406535863c9f3b62bf646bdfc46d4